train_diffute_v2.py

```
@@ -0,0 +1,498 @@
+
+    vae = AutoencoderKL.from_pretrained(
+        args.pretrained_model_name_or_path,
+        subfolder="vae",
+    )
+
+    unet = UNet2DConditionModel.from_pretrained(
+        args.pretrained_model_name_or_path,
+        subfolder="unet",
+    )
+
+    processor = TrOCRProcessor.from_pretrained("microsoft/trocr-large-printed")
```

Ruff (0.8.2)

Local variable `processor` is assigned to but never used

Remove assignment to unused variable `processor`

(F841)